### PR TITLE
Prioritize `entity_name` over `root`

### PR DESCRIPTION
**Pull request by gekola**

### CODE REVIEW

File: spec/api_models_spec.rb [hunk 1]
```diff
@@ -305,9 +320,15 @@ def app
...
   end
 
-  it 'includes an id equal to the model name' do
+  it 'includes an id equal to the model name (defined with root)' do
     get '/swagger_doc/thing_with_root'
     result = JSON.parse(last_response.body)
     expect(result['models']['thing']['id']).to eq('thing')
   end
+
+  it 'preferes entity_name for id over root' do
```
Comment (dblock) [inline spec/api_models_spec.rb]: preferes -> prefers

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 * Your contribution here.
 
+* [#322](https://github.com/ruby-grape/grape-swagger/pull/322): Entity's `entity_name` takes predence over root  - [@gekola](https://github.com/gekola)
```
Comment (dblock) [inline CHANGELOG.md]: Add a period at the end ;)

**Comment by dblock** [comment]
:+1: Just minor spelling fixes above please.

**Comment by gekola** [comment]
@dblock, I've fixed spelling and rebased the PR on master.

**Pull request merged**